contentcuration/contentcuration/utils/publish.py

```diff
@@ -673,18 +707,31 @@ def process_assessment_metadata(ccnode, kolibrinode):
         }
     )
 
+    return exercise_data
+
+
+def create_kolibri_assessment_metadata(ccnode, kolibrinode):
+    assessment_items = ccnode.assessment_items.all().order_by("order")
+    assessment_item_ids = [a.assessment_id for a in assessment_items]
+    randomize, _, mastery_model = _get_exercise_data_from_ccnode(
+        ccnode, len(assessment_item_ids)
+    )
```

Comment:
@rtibbles i meant these lines and [this](https://github.com/learningequality/studio/pull/5103/commits/95a5c323bb5efdd30a20a12bffa6743c8546aaf3#r2201229945) that seem to do pretty much the same thing.